### ISSUE TITLE
DVC status of locked files

### ISSUE THREAD

**Issue by prihoda** [iterative/dvc#895]
Hi guys, the locking feature is very useful! Could you mark locked files in `dvc status` or just hide them? It would be useful especially for locked callback stages as mentioned in #751.

**Comment by efiop** [comment]
Hi @prihoda !

I'm actually working on enhancing results of `locking` in other commands. I will take a look at `status` today as well. Expect it to be merged and released today/tomorrow.

Thank you for your amazing feedback!

--Ruslan

**Comment by efiop** [comment]
`dvc status` now doesn't output dependencies of locked dvc files and it also prints a warning that it is locked, since the implications of it are pretty harsh. It is going to be released in 0.13.0 later today.

Thanks,
Ruslan

**Comment by prihoda** [comment]
Hey guys, is it expected behavior that locked files without dependencies are printed in status output?

```bash
dvc run -o test.txt touch test.txt
## Running command:
##	touch test.txt
## Warning: File/directory 'test.txt' is empty.
## Saving 'test.txt' to cache '../.dvc/cache'.
## Saving information to 'test.txt.dvc'.

dvc status test.txt.dvc 
## test.txt.dvc

dvc lock test.txt.dvc 
## Saving information to 'test.txt.dvc'.

dvc status test.txt.dvc 
## Warning: DVC file 'test.txt.dvc' is locked. Its dependencies are not going to be shown in the status output.
## test.txt.dvc <------ should this be printed?
```

**Comment by efiop** [comment]
Hi @prihoda !

Great catch! That is definitely a bug. Created https://github.com/iterative/dvc/issues/1381 for it.

Thank you!

**Comment by efiop** [comment]
Hi @prihoda !

Fix for #1381 was released in 0.21.3 as well. Please upgrade :slightly_smiling_face: Thanks again for all the feedback!
